prometheus-exporters/pvc-exporter/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: keppel.eu-de-1.cloud.sap/ccloud/block-pvc-scanner
+  tag: 7c34a1ef
+
+namespace: kube-monitoring
+metrics:
+  port: 8848
+
+  # Name of the Prometheus by which this exporter is scraped.
+  prometheus: collector-kubernetes
+
+resources:
+  requests:
+    memory: 40Mi
+    cpu: 10m
+  limits:
+    memory: 100Mi
+    cpu: 20m
```

Comment:
I think that is too low, the process might not be able to start. Please set to either 100m or 200m.